Add a community strategy sub-page and provide a brief update on works so far (#560)
jupyterhub/team-compass: docs/team/community-strategy.md

# Community Strategy

This page contains information regarding activities undertaken and community strategies developed by the Community Strategic Lead.
[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.

> This role and related activities are funded by a CZI EOSS DEI grant, administered by NumFOCUS.

Reviewer: ```suggestion
This role and related activities are funded by [a CZI EOSS DEI grant](https://drive.google.com/file/d/124LWcFe8Hq2n_3l4jAgDgJ1lpXRzV1Kc/view?usp=sharing), administered by [NumFOCUS](https://numfocus.org).
```

```{admonition} Work in Progress
The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
```

> Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.

Reviewer: The PyData theme will soon release support for lightly auto-formatting github issue/PR links, so we could just put in the raw link if we like: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services

> Some information is being shared in [Issue #536](https://github.com/jupyterhub/team-compass/issues/536), which will be migrated here over time.

Author: ```suggestion
<!-- Using a raw link here as a PyData Sphinx theme release will format them for us: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services -->
Some information is being shared in https://github.com/jupyterhub/team-compass/issues/536, which will be migrated here over time.
```

Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.

> ## Community Strategy Updates

Reviewer: Eventually we probably want a dedicated page for these (e.g. a "Changelog" for the CSL position) but I think it's fine here for now

> ### 2022-10-07: 2 month strategy so far

Author: Nope, this is a "Sarah can't write down dates properly" date

> ### 2022-10-07: 2 month strategy so far

Author: ```suggestion
### 2022-09-07: 2 month strategy so far
```

> ### 2022-10-07: 2 month strategy so far

Author: Fixed in [`f3992b0` (#560)](https://github.com/jupyterhub/team-compass/pull/560/commits/f3992b0f7772c54d295b580c29f3b3ce5c7826b6)

> One strategy is to develop onboarding pathways through Outreachy,then into Contributor in Residence, and finally as a fully-fledged community member.

Reviewer: Let's flesh this out in a future update - we can brainstorm via these updates, and then encode things in the team compass via more focused PRs

[Sarah Gibson](https://github.com/sgibson91) is currently serving in this role as of August 2022, for a period of two years.

This role and related activities are funded by [a CZI EOSS DEI grant](https://drive.google.com/file/d/124LWcFe8Hq2n_3l4jAgDgJ1lpXRzV1Kc/view?usp=sharing), administered by [NumFOCUS](https://numfocus.org).

```{admonition} Work in Progress
The work represented on this page is under active development and we will update information as soon as we are able to as we iterate.
```

<!-- Using a raw link here as a PyData Sphinx theme release will format them for us: https://pydata-sphinx-theme.readthedocs.io/en/latest/user_guide/theme-elements.html#link-shortening-for-git-repository-services -->
Some information is being shared in https://github.com/jupyterhub/team-compass/issues/536, which will be migrated here over time.
Additionally, you can find information on other related activities via the [![csl-label](https://img.shields.io/github/labels/jupyterhub/team-compass/community-strategic-lead)](https://github.com/jupyterhub/team-compass/issues?q=label%3Acommunity-strategic-lead) label.

## Community Strategy Updates

### 2022-09-07: 2 month strategy so far

> One strategy is to develop onboarding pathways through Outreachy,then into Contributor in Residence, and finally as a fully-fledged community member.

Author: ```suggestion
One strategy is to develop onboarding pathways through Outreachy, then into Contributor in Residence, and finally as a fully-fledged community member.
```
